datarequest.py

```diff
@@ -37,20 +38,26 @@
            'api_datarequest_data_ready']
 
 
-DRCOLLECTION      = "home/datarequests-research"
 SCHEMACOLLECTION  = constants.UUSYSTEMCOLLECTION + "/datarequest/schemas/youth"
+SCHEMA_FILENAME   = "schema.json"
+UISCHEMA_FILENAME = "uischema.json"
 
 GROUP_DM          = "datarequests-research-datamanagers"
 GROUP_DMC         = "datarequests-research-data-management-committee"
 GROUP_BOD         = "datarequests-research-board-of-directors"
 
-SCHEMA_FILENAME   = "schema.json"
-UISCHEMA_FILENAME = "uischema.json"
-DR_FILENAME       = "datarequest.json"
-PR_REV_FILENAME   = "preliminary_review.json"
-DM_REV_FILENAME   = "datamanager_review.json"
-EVAL_FILENAME     = "evaluation.json"
-ASSIGN_FILENAME   = "assignment.json"
+DRCOLLECTION      = "home/datarequests-research"
+DR                = "datarequest"
+PR_REV            = "preliminary_review"
+DM_REV            = "datamanager_review"
+REV               = "review"
+ASSIGN            = "assignment"
+EVAL              = "evaluation"
```

Comment:
EVAL -> EVALUATION
REV -> REVIEW
etc.

No need to use abbreviation. I would read REV as revision